aws/resource_aws_lb_test.go

```diff
@@ -980,6 +984,7 @@ resource "aws_route_table_association" "a" {
 resource "aws_lb" "test" {
   name            = "%s"
   load_balancer_type = "network"
+  enable_cross_zone_load_balancing = true
```

Comment:
See my note above -- we'll probably want to just copy paste the handling of the new attribute to a new test and config or at least pass the true/false value for this attributes is a directive into the above config function and use `enable_cross_zone_load_balancing = %v` 😄